tests/bledtm/syscfg.yml

```diff
@@ -0,0 +1,33 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+syscfg.defs:
+    DTM_RF_CHANNEL:
+        description: RF channel used by transmitter
+        value: 0
+        range: 0..39
+    DTM_PAYLOAD_LENGTH:
+        description: Length of payload of the test reference packets
+        value: 100
+        range: 0..255
+    DTM_PAYLOAD_TYPE:
```

Comment:
since name is bledtm I would add here DTM_ROLE: so we could also configure second board on RX.
On RX side we could have a timer which would stop DTM. Once timer is stopped we could print the result and start RX and time again

Reply:
agree, but we can add this syscfg when we have RX role, for now it's redundant

Reply:
We need it to verify TX role :)

Reply:
To be clear, we need RX role, not only syscfg

Reply:
sure, but we can verify using blehci as usual and add RX role later - I do not think that lack of RX support in this app should be considered a blocker, someone can add it later if needed.